src/ts/components/fullscreentogglebutton.ts

```diff
@@ -20,28 +20,32 @@ export class FullscreenToggleButton extends ToggleButton<ToggleButtonConfig> {
   configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    let fullscreenStateHandler = () => {
-      if (player.getViewMode() === player.exports.ViewMode.Fullscreen) {
-        this.on();
-      } else {
-        this.off();
-      }
+    let isFullScreenAvailable = () => {
```

Comment:
You could use `const` for functions that won't change

Reply:
53e0f15

Reply:
💪